.circleci/config.yml

```diff
@@ -11,6 +11,8 @@ jobs:
             - browser-tools/install-chrome
             - browser-tools/install-chromedriver
             - checkout
+            # CircleCI seems to mess with master ref which causes lint:version check to fail
+            - run: git branch -f master origin/master
```

Comment:
I'm not sure about this. I don't really understand why we need `vertioner` to begin with though.

Reply:
Yeah agree, this is not ideal. But CircleCI seems to map local master branch to the current PR or something. Have been thinking about a better solution - will keep looking.

Reply:
This has been removed now.